docs/local-metadata.md

```diff
@@ -0,0 +1,54 @@
+Local Metadata
+==============
+
+Local metadata can be saved in `/var/lib/fwupd/metadata.d` or `/usr/share/fwupd/metadata.d` which is
+scanned at daemon startup. This can be used to add site-specific BKC tags to existing metadata
+stores. For instance:
```

Comment:
Please tell me this isn't going to turn into Acme Corp distributing debs or RPMs with this information...

Reply:
I think the plan is for the Google management agent thing to write files into /var/lib/fwupd/metadata.d as required.